src/pages/instances/Events.tsx

```diff
@@ -77,10 +104,14 @@ const Events: FC = () => {
       const ws = new WebSocket(wsUrl);
       ws.onopen = () => {
         setEventWs(ws);
+        setWsOpenTime(getCurrentTime());
       };
       ws.onclose = () => {
         setEventWs(null);
       };
+      ws.onerror = () => {
+        setEventWs(null);
+      };
```

Comment:
Maybe we can extract `setEventWs(null);` into a reusable `resetEventWs` function, ad use as `onclose`, `onerror` handler and everywhere else we need to reset the event ws to null?

Reply:
Having a `resetEventWs` function that can be reused (and needs to be reused already) doesn't seem this absurd complication to me, but again - YMMV, so as you prefer.